packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -271,6 +292,18 @@ class NewDisk extends Component {
     resourceSet => get(resourceSet, 'limits.disk.available')
   )
 
+  _getSrNotOnSameHost = createSelector(
+    () => this.state.sr,
+    () => this.props.srs,
+    () => this.props.vdis,
```

Comment:
What if the VM has an existing vdis contained in different hosts ?

Reply:
The warning message will be displayed but the VM can't be started. 
@pdonias what should we do in this case ?